packages/loot-core/src/server/spreadsheet/new/parser.js

```diff
@@ -132,6 +132,7 @@ function parseAnd(state) {
 function parseNot(state) {
   let left = parseCompare(state);
   while (skipValue(state, types.TOKEN_OPERATOR, 'not')) {
+    // eslint-disable-next-line no-unused-vars
     const right = parseCompare(state);
```

Comment:
Keeping this unused var for consistency with the neighboring functions?

Reply:
No, actually kept this because I wasn't sure if `parseCompare` mutates state or not. So left it in just to be safe.

I'm planing to do a second pass of all the `eslint-disable` lines sometime later. Will prune it then if I understand that it's unnecessary.